libshvvisu/src/timeline/graph.cpp

```diff
@@ -1102,8 +1102,7 @@ void Graph::makeLayout(const QRect &pref_rect)
 		ch->m_layout.yAxisRect.setWidth((m_style.isYAxisVisible()) ? u2px(m_style.yAxisWidth()) : 0);
 
 		widget_height += ch->m_layout.graphAreaRect.height();
-		if(i > 0)
-			widget_height += u2px(m_style.channelSpacing());
+		widget_height += u2px(m_style.channelSpacing());
```

Comment:
```suggestion
if (i > 0)
		widget_height += u2px(m_style.channelSpacing());
```

Reply:
Mas pravdu, ta podminka tam patri, ale podle me spis takhle:
`if (i + 1 < visible_channels.count()) `
ne?

Reply:
a neni to to samy?

Reply:
Asi neni, bud se prida mezera vsude krome prvniho nebo vsude krome posledniho.

Reply:
No to jo, ale nejde jen o vysledny soucet. Promenna se pouziva uz v tom cyklu, kde se nastavuje pozice kanalu 
radek 1094
```
ch->m_layout.graphAreaRect.moveTop(widget_height);
```
Takze v tom tvem pripade bude chybet mezera mezi prvnim a druhym kanalem a naopak bude mezera mezi poslednim kanalem a osou X.